rheem-core/src/main/java/org/qcri/rheem/core/util/fs/FileSystems.java

```diff
@@ -88,7 +88,7 @@ public static OptionalLong getFileSize(String fileUrl) {
 
             // Look for Spark-like directory structure.
             if (children.stream().anyMatch(child -> child.endsWith("_SUCCESS"))) {
-                return children.stream().filter(child -> child.matches(".*/part-\\d+")).collect(Collectors.toList());
+                return children.stream().filter(child -> child.matches(".*/part-[m|r|M|R|-]{0,2}\\d+")).collect(Collectors.toList());
```

Comment:
Is this the output format of Giraph/Hadoop? In that case, we might need to extend the command above.